src/Command/Vote.js

```diff
@@ -2,12 +2,12 @@ const Command = require('../Command');
 
 module.exports = Command.extend({
     commandName: 'vote',
+    config: null,
+    dependencies: {
```

Comment:
This isn't necessary as dependencies are merged with their parent class. Please remove it.